pycbc/inference/models/relbin.py

```diff
@@ -169,10 +171,17 @@ def __init__(
             variable_params, data, low_frequency_cutoff, **kwargs
         )
 
+        # If the waveform handles the detector response internally, set
+        # self.det_response = True
+        self.det_response = False
+        if self.static_params['approximant'] in fd_det_sequence:
+            self.det_response = True
```

Comment:
The variable naming seems backwards to me as it could be read that the likelihood *will* handle the detector response if this is set.